includes/apple-exporter/builders/class-components.php

```diff
@@ -691,15 +691,28 @@ private function group_body_components( $components ) {
 		 * All components after the cover must be grouped to avoid issues with
 		 * parallax text scroll.
 		 */
+		$conditional = array();
+		if ( ! empty( $theme->get_value( 'body_background_color_dark' ) ) ) {
+			$conditional = array(
+				'conditional' => array(
+					'backgroundColor' => $theme->get_value( 'body_background_color_dark' ),
+					'conditions'      => array(
+						'preferredColorScheme' => 'dark',
+					),
+				),
+			);
+		}       
 		$regrouped_components = array(
 			'role'       => 'container',
 			'layout'     => array(
 				'columnSpan'           => $theme->get_layout_columns(),
 				'columnStart'          => 0,
 				'ignoreDocumentMargin' => true,
 			),
-			'style'      => array(
-				'backgroundColor' => $theme->get_value( 'body_background_color' ),
+			'style'      => (
+				array(
+					'backgroundColor' => $theme->get_value( 'body_background_color' ),
+				) + $conditional
 			),
```

Comment:
I mulled this over, and I'd like to use `array_merge` in all instances rather than `$array1 + $array2`. There are two reasons for this: 1. `array_merge` is more readable. It's more clear what's happening. 2. Using `array_merge` is more akin to the behavior of an object rest spread in JavaScript, where if keys exist, it will overwrite them. If a key exists and you do `$array1 + $array2` it will not overwrite the original key, despite that being the intention, and that could be a difficult bug to chase down. Here is the same code block rewritten to use `array_merge` instead:
```suggestion
			'style'      => array_merge(
				array(
					'backgroundColor' => $theme->get_value( 'body_background_color' ),
				),
				$conditional
			),
```